### ISSUE TITLE
Guice + endpoints v2 example?

### ISSUE THREAD

**Issue by AlistairB** [GoogleCloudPlatform/java-docs-samples#838]
https://cloud.google.com/endpoints/docs/frameworks/legacy/v1/java/migrating mentions guice can be incorporated with:

```java
public class FooEndpointsModule extends EndpointsModule {
  @Override
  public void configureServlets() {
    bind(TestEndpoint.class).toInstance(new TestEndpoint());
    configureEndpoints("/_ah/api/*", ImmutableList.of(TestEndpoint.class));
  }
}
```

However, this doesn't explain what would go in `web.xml` or configuring a `GuiceServletContextListener`.

Can https://github.com/GoogleCloudPlatform/java-docs-samples/tree/master/appengine/endpoints-frameworks-v2/migration-example add guice to be a more complete and realistic example?

I am unable to figure out how to do this. It seems others are also struggling with this https://stackoverflow.com/questions/45760488/example-for-using-guice-with-endpoints-frameworks-for-app-engine-version-2

**Comment by lesv** [comment]
@frankyn Care to comment?

**Comment by frankyn** [comment]
Hi @AlistairB, 

Thanks for filing an issue! I have WIP sample for Guice + Endpoints you can take a look at [here](https://github.com/GoogleCloudPlatform/java-docs-samples/tree/endpoints-update/appengine-java8/endpoints-v2-guice). Please let me know if this helps.

**Comment by henriquefm** [comment]
> Thanks for filing an issue! I have WIP sample for Guice + Endpoints you can take a look at here. Please let me know if this helps.

Thanks for that example, it will be helpful. I'm getting a 404 at that link though, this one works: 
https://github.com/GoogleCloudPlatform/java-docs-samples/tree/master/appengine-java8/endpoints-v2-guice

**Comment by frankyn** [comment]
It was merged in during PR#842. It's not located where you found it. :)
